airflow/example_dags/example_branch_operator_decorator.py

```diff
@@ -112,7 +112,7 @@ def some_ext_py_task():
     #       Run the example a second time and see that it re-uses it and is faster.
     VENV_CACHE_PATH = tempfile.gettempdir()
 
-    @task.branch_virtualenv(requirements=["numpy~=1.24.4"], venv_cache_path=VENV_CACHE_PATH)
+    @task.branch_virtualenv(requirements=["numpy~=1.26.0"], venv_cache_path=VENV_CACHE_PATH)
```

Comment:
Note: needed to change this as else the example is broken and venv does not work with Python 3.12

Reply:
Yep. Separating is a good idea.

Reply:
Separated to #43653